src/install.rst

```diff
@@ -184,6 +184,29 @@ Set up a Nextstrain runtime
             .. include:: snippets/nextstrain-setup-conda.rst
 
 
+   .. group-tab:: Singularity (Apptainer)
```

Comment:
I was reviewing the rendered docs and realized that we're missing this tab in the "[Try running Augur and Auspice](https://nextstrain--185.org.readthedocs.build/en/185/install.html#try-running-augur-and-auspice)" section of the install page. Singularity is also missing from the "[Managing an installation](https://nextstrain--185.org.readthedocs.build/en/185/guides/manage-installation.html)" page. Let's add it there too?

Reply:
Not sure what to add under "Troubleshoot a broken installation". I tried setting it up on `rhino` to get a sense of what to put here, but that was [unsuccessful](https://github.com/nextstrain/docs.nextstrain.org/pull/185/files#r1476544501)). Any ideas?

Reply:
> Not sure what to add under "Troubleshoot a broken installation"

I think something relatively generic like what the Conda tab uses would be ok.

Reply:
Added in [force-push](https://github.com/nextstrain/docs.nextstrain.org/compare/0961b9eb3564f7f776c1791c8a27e3e5e5f29567..8b29c2f846358612ed461d5cf2456ec9de4e56ae)